source/_integrations/filter.markdown

```diff
@@ -58,6 +58,7 @@ sensor:
         window_size: "00:05"
         precision: 2
 ```
+The example above will create two new entities: sensor.filtered_realistic_humidity and sensor.filtered_realistic_temperature 
```

Comment:
I honestly thin this isn't correct. Entity IDs are not guaranteed at all, and above all can be customized in certain cases.

Nevertheless, thanks for being willing to contribute 👍 

../Frenck